forms/gridfield/GridFieldExportButton.php

```diff
@@ -115,12 +115,14 @@ protected function getExportColumnsForGridField(GridField $gridField) {
 	 * Generate export fields for CSV.
 	 *
 	 * @param GridField $gridField
-	 * @return array
+	 * @return string
 	 */
 	public function generateExportFileData($gridField) {
 		$separator = $this->csvSeparator;
 		$csvColumns = $this->getExportColumnsForGridField($gridField);
-		$fileData = '';
+
+		$stream = fopen('php://memory', 'w');
```

Comment:
> can't use try {} finally {} as that's php 5.6 and 3.x has to support 5.3.3+

Oh darn, that's my bad. =( You will need to mock it with a try {} catch {} then.